### DISCUSSION TITLE
option to sort by most recent beatmaps

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#19926] [Ideas]
in the osu direct screen in standard you had an option to sort by most recent beatmaps (within your other filter criteria), which i liked to discover new beatmaps i hadn't heard/seen/played before. I kinda wish lazer/the website beatmap listing had this option as well, as i found it to be quite useful to discover new beatmaps every week or two.

**Comment by iSimpFor** [comment]
i think you can just sort by ranked for that
